cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -222,30 +232,11 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
     # Apply new config
```

Comment:
REQUIRED : please add assert before reconfiguration in order to see that values are really different.